apps/web/src/routes/_content/courses/index.tsx

```diff
@@ -89,31 +112,41 @@ export const CourseCard = ({ course }: { course: JoinedCourse }) => {
           alt={course.name}
           className="md:hidden rounded-md w-[124px] object-cover [overflow-clip-margin:_unset] object-center"
         />
-        <div className="flex flex-col">
-          <span className="max-md:flex flex-col mb-1.5 md:mb-2 line-clamp-3 group-hover:line-clamp-2 font-medium leading-[120%] tracking-015px md:desktop-h6 text-darkOrange-5 max-md:h-full align-top">
+        <div className="flex flex-col md:gap-2">
+          <span className="max-md:flex flex-col md:mb-2 line-clamp-2 lg:group-hover:line-clamp-2 font-medium leading-[120%] tracking-015px md:desktop-h6 text-white md:align-top mb-2 lg:mb-0">
```

Comment:
line clamp don't seem to be working
![image](https://github.com/user-attachments/assets/8217fb67-61d0-4a7f-8714-3cfa76786264)